continuity/manager_relaystate.go

```diff
@@ -0,0 +1,151 @@
+package continuity
```

Comment:
This file is almost a 1:1 copy of manger_cookie. Can you please remove it and use manager_cookie instead? If you need to parametrize things, feel ree to adjust the manager_cookie file :)

Reply:
Done! We now use the manager_cookie and make the use of the RelayState configurable :)